.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Please feel free to remove whatever sections/lines in this aren't relevant. -->
+
+## Title Line Template: [Website] - [UI components]: [Brief description.]
+<!-- Use the title line as the title of your pull request, then delete these lines. 
+
+Website: Issues that impact standards.usa.gov look, feel or functionality.
+UI components: Issues that impact the look, feel or functionality of the standards themselves.
+
+-->
+
+## Description
+
+Include a high-level description of the issue. What did you expect to happen? What happened instead? What would you like to see changed?
+
+Include any benefits, challenges, or considerations. This can be short and sweet.
+
+## Steps to reproduce the issue
+
+1. Step one
+2. Step two
+3. Step three
+4. etc.
```

Comment:
Might recommend replacing _etc._ with _And so on_ OR _And so forth_. (Reasoning: Our content guide recommends against using _etc._ because it can feel jarring.)